src/rust/ide/src/model/suggestion_database.rs

```diff
@@ -427,9 +102,13 @@ impl SuggestionDatabase {
                 Err(err)  => { error!(logger,"Discarded invalid entry {id}: {err}"); },
             }
         }
+        //TODO[ao] this is temporary solution. Eventually we should gather examples from
```

Comment:
missing "a" before "temporary. Also missing big letter on the beginning of first sentence. Missing comma after "Eventually".